ui/src/store/modules/user.js

```diff
@@ -415,4 +469,16 @@ const user = {
   }
 }
 
+export function newVersionAvailable () {
```

Comment:
Do you think an "Acknowledged" kind of option makes sense for version notification, otherwise it will keep reminding time to time or on every login. If someone wants to stay on a particular release, they see this notification everytime. @vishesh92